modules/core/module.go

```diff
@@ -197,7 +197,9 @@ func (AppModule) ConsensusVersion() uint64 { return 7 }
 
 // BeginBlock returns the begin blocker for the ibc module.
 func (am AppModule) BeginBlock(ctx context.Context) error {
-	ibcclient.BeginBlocker(sdk.UnwrapSDKContext(ctx), am.keeper.ClientKeeper)
+	if err := ibcclient.BeginBlocker(sdk.UnwrapSDKContext(ctx), am.keeper.ClientKeeper); err != nil {
```

Comment:
It's OK to catch the error and log it, but we cannot return an error here as it would not let the block finalize and subsequently halt the chain